changelogs/fragments/3450-callback_opentelemetry-exception_handling.yml

```
@@ -0,0 +1,3 @@
+minor_changes:
```

I think this qualifies as a bugfix.
```suggestion
bugfixes:
```